### ISSUE TITLE
Wizdom.xyz provider stopped working

### ISSUE THREAD

**Issue by Shasoosh** [morpheus65535/bazarr#1870]
Wizdom.XYZ stopped downloading subtitles. 

**To Reproduce**
1. Add wizdom is a single provider (Hebrew language)
2. Do a manual search for "Encanto" (https://www.imdb.com/title/tt2953050/)
3. No results return

**Expected behavior**
Subtitles should be found.

**Software (please complete the following information):**
 - Bazarr: 1.0.4
 - OS: DSM 7.1

**Additional context**
Looking at the [wizdom script](https://github.com/morpheus65535/bazarr/blob/e305aad597e67f8da5cfd298ce6039e8e59642ef/libs/subliminal_patch/providers/wizdom.py) it seems like a GET request to https://json.wizdom.xyz/tt2953050.json should work but this returns an error:
```
"QUERY: SELECT *,UPDATE as sdates FROM subs WHERE IMDB_ID='tt2953050' ORDER BY SEASON,EPISODE ASC
ERROR: Table 'subs.subs' doesn't exist in engine"
```

**Comment by morpheus65535** [comment]
I'm getting blocked on first request to the new API using my browser: `{"statusCode":429,"message":"ThrottlerException: Too Many Requests"}`

**Comment by Shasoosh** [comment]
I’m not getting this.

You’re getting this from just clicking on the url? 

https://wizdom.xyz/api/releases/tt2953050

**Comment by morpheus65535** [comment]
I was but not anymore... seams strange... I'll see what I cna do with this.

**Comment by morpheus65535** [comment]
Should be fixed in upcoming beta. Can you test and confirm please?

**Comment by Shasoosh** [comment]
np. Thanks!

**Comment by Shasoosh** [comment]
Working, thanks!

**Comment by omrij** [comment]
Semi related to this issue
When the subtitle is found, all is working OK
Like this one
https://wizdom.xyz/api/releases/tt2953050
But when no subtitle is found
like this one
https://wizdom.xyz/api/releases/tt17043230
It's disabling the provider for 10 minutes

`Throttling wizdom for 10 minutes, until 22/12/04 22:48, because of: HTTPError. Exception info: '500 Server Error: Internal Server Error for url: https://wizdom.xyz/api/releases/tt17043230'`

**Comment by morpheus65535** [comment]
@omrij open a new issue please.
